### PR TITLE
install latest CellRegMap from github in image

### PR DESCRIPTION
**Pull request by annacuomo**
I need to use CellRegMap functions that are not yet released to PyPI, so installing directly from github (but not 100% sure of the syntax!). Thanks!

### CODE REVIEW

File: images/cellregmap/Dockerfile [hunk 1]
```diff
@@ -14,7 +15,7 @@ RUN apt-get update && \
     # The chi2comb-py Python package depends on the chi2comb C++ library.
     curl -fsSL https://raw.githubusercontent.com/limix/chi2comb/master/install | bash && \
     pip3 install --no-cache-dir \
-        cellregmap==${VERSION} \
+        cellregmap==git+https://github.com/annacuomo/CellRegMap \
```
Comment (illusional) [inline images/cellregmap/Dockerfile:18]:
```suggestion
        git+https://github.com/annacuomo/CellRegMap \
```

File: images/cellregmap/Dockerfile [hunk 2]
```diff
@@ -1,10 +1,11 @@
-ARG DRIVER_TAG="latest"
+# ARG DRIVER_TAG="latest"
+ARG DRIVER_TAG="dev"
```
Comment (illusional) [inline images/cellregmap/Dockerfile:2]: This is the analysis-runner DRIVER_TAG, uses below, don't change this.

**Review by illusional — CHANGES_REQUESTED** [review]

File: images/cellregmap/Dockerfile (hunk 2 above)
Comment (annacuomo) [reply images/cellregmap/Dockerfile:2]: ok, where am i adding a dev tag then?

Comment (illusional) [reply images/cellregmap/Dockerfile]: On deploy

**Review by illusional — APPROVED** [review]

**Pull request merged**